packages/server/lib/cloud/api.ts

```diff
@@ -330,6 +331,12 @@ module.exports = {
         })
       })
     })
+    .then(async (result) => {
+      // TODO: Get url for the protocol code and pass it down to download
```

Comment:
Suggest namespacing all the TODOs so we can easily verify that all the TODOs have been completed.

```suggestion
      // TODO(protocol): Get url for the protocol code and pass it down to download
```

Reply:
good call.